tests/timezones/tests.py

```diff
@@ -123,7 +123,8 @@ def test_aware_datetime_in_other_timezone(self):
     @skipIfDBFeature('supports_timezones')
     def test_aware_datetime_unsupported(self):
         dt = datetime.datetime(2011, 9, 1, 13, 20, 30, tzinfo=EAT)
-        with self.assertRaises(ValueError):
+        msg = r'%s backend does not support timezone-aware datetimes when USE_TZ is False.' % connection.vendor
```

Comment:
I'd use `assertRaisesMessage()` with `msg = 'backend does not support timezone-aware datetimes when USE_TZ is False.'`. There's a small chance that messages from third-party backends might not match the `connection.vendor` convention and I don't think it's a critical part of the assertion.